packages/drivers/odsp-driver/src/compactSnapshotParser.ts

```diff
@@ -175,6 +176,10 @@ function readTreeSection(node: NodeCore) {
 			snapshotTree.unreferenced = true;
 		}
 
+		if (records.groupId !== undefined) {
```

Comment:
You can remove the changes in compactSnapshotParser.ts and compactSnapshotWriter.ts which are on download path/snapshot parsing code which I am handling here: https://github.com/microsoft/FluidFramework/pull/19494